app/services/idv/usps_mail.rb

```diff
@@ -0,0 +1,36 @@
+module Idv
+  class UspsMail
+    def initialize(current_user)
+      @current_user = current_user
+      @max_events = 4
+    end
+
+    def mail_spammed?
+      any_mail_events? && correct_length && updated_within_last_month
+    end
+
+    private
+
+    def find_user_events(event_type, user = @current_user)
+      Event.order('updated_at DESC').
+        limit(@max_events).
+        where(user_id: user.id, event_type: Event.event_types[event_type])
+    end
+
+    def user_mail_events
+      find_user_events(:usps_mail_sent, @current_user)
+    end
+
+    def correct_length
+      user_mail_events.count == 4
+    end
+
+    def any_mail_events?
+      user_mail_events.any?
+    end
+
+    def updated_within_last_month
+      user_mail_events.last.updated_at > 1.month.ago
```

Comment:
ditto, the `1.month.ago` should definitely be a constant and we also probably want to makae it configurable via env vars